Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -765,13 +765,14 @@ - (WMFLayoutEstimate)collectionView:(UICollectionView *)collectionView estimated
         case WMFFeedDisplayTypePageWithPreview: {
             WMFArticle *article = [self articleForIndexPath:indexPath];
             WMFArticlePreviewCollectionViewCell *cell = [self placeholderCellForIdentifier:[WMFArticlePreviewCollectionViewCell wmf_nibName]];
-            
+            [self.view insertSubview:cell atIndex:0];
```

Comment:
Intereresting! We may want an assert that cell's superview is self.view as a reminder in case we try to remove this in the future...